src/app/loaders/express/error-handler.ts

```diff
@@ -22,6 +22,14 @@ const isHTTPLikeError = (
   return types.isNativeError(err) && 'statusCode' in err
 }
 
+// 404 since no middleware responded
+export const catchNonExistentRoutesMiddleware: RequestHandler = function (
```

Comment:
ha, we probably missed this file while converting all functions to arrow functions. can fix another time